bank2ynab.conf

```diff
@@ -910,6 +910,38 @@ Header Rows = 0
 Input Columns = Date,skip,Outflow,Inflow,Payee,Memo,skip
 Date Format = %d %b %Y
 
+[SG Standard Chartered Savings Account]
+# Issue: 458
+Source Filename Pattern = ACC_TXN_History_\d{14}
+Source Filename Extension = .xls
+Use Regex for Filename = True
+Source CSV Delimiter = ,
```

Comment:
You can omit entries that are the same as the default values to simplify the config, e.g. `Source CSV Delimiter = ,`. Check the defaults at the top of the file.